JVAlertController/JVAlertController/JVPopoverPresentationController.m

```diff
@@ -26,14 +26,92 @@
 #import <objc/runtime.h>
 #import <UIKit/UIKit.h>
 
-@interface JVPopoverPresentationController : NSObject
+@interface JVPopoverPresentationController : NSObject <UIPopoverControllerDelegate>
 @property (nonatomic, strong) UIView *sourceView;
 @property (nonatomic) CGRect sourceRect;
 @property (nonatomic, strong) UIBarButtonItem *barButtonItem;
+@property (nonatomic, weak) id<UIPopoverPresentationControllerDelegate> delegate;
```

Comment:
No. For me the backport of ```UIAlertController``` provides the opportunity to work with this iOS 8 class on former iOS versions and thus I need all iOS 8 protocols and related infrastructure available as they would on iOS8. If the code must compile seamlessly on iOS 8 (only with ```UIAlertViewController```) and iOS 7 (with ```JVAlertController```) it must use the iOS 8 protocol delegate.

It seems as if the ```UIPopoverPresentationControllerDelegate``` protocol definition is available as soon as you compile with the iOS 8 SDK because its headers are loaded and all definitions compiled into your binary. So you can use it even with iOS 7.